### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by fokoz** [Kishanjvaghela/react-native-cardview#35]
**Describe the bug**
build failed as follow log:

```
Execution failed for task ':app:transformClassesWithDexBuilderForDebug'.
> com.android.build.api.transform.TransformException: com.android.builder.dexing.DexArchiveBuilderException: com.android.builder.dexing.DexArchiveBuilderException: Failed to process /node_modules/react-native-cardview/android/build/intermediates/intermediate-jars/debug/classes.jar
```

with setting 
`multiDexEnabled true`

**To Reproduce**
1. run `react-native run-android`

**Smartphone (please complete the following information):**
 - Device: Vivo
 - OS: Android
 - Version 6.0.1

**Comment by jim20june1989** [comment]
Execution failed for task ':app:transformClassesWithDexBuilderForDebug'.
> com.android.build.api.transform.TransformException: com.android.builder.dexing.DexArchiveBuilderException: com.android.builder.dexing.DexArchiveBuilderException: Failed to process /node_modules/react-native-cardview/android/build/intermediates/intermediate-jars/debug/classes.jar

Am facing the same issue

**Comment by Kishanjvaghela** [comment]
Can you delete
`node_modules/react-native-cardview/android/build/`
And try again?

**Comment by jim20june1989** [comment]
Hello Kishan Vaghela,

Thanks for your reply. As you suggested I proceeded and came up with this
error

kishanjvaghela/cardview/RNCardView.java:4: error: package
android.support.v7.widget does
not exist
import android.support.v7.widget.CardView;

Can you let me know what could be the problem?

*Thanks,*
*Jimmy James *
*Software Engineer*

First Floor, Idasseril Chambers, Link Park,

Pottakuzhy Road, Vaduthala,

Cochin, Kerala – 682012

Ph: +91 702 501 1144 <+91+702+501+1144>

Email: contact@emvigotech.com
<https://www.emvigotech.com/contactus/contact@emvigotech.com>

**Comment by Kishanjvaghela** [comment]
Open android studio and try to sync gradle.

**Comment by mattop1709** [comment]
The build has succeed after migrating to AndroidX.

**Comment by Kishanjvaghela** [comment]
Fixed here. I will push to npm https://github.com/Kishanjvaghela/react-native-cardview/pull/51

**Comment by Kishanjvaghela** [comment]
Published to v2.0.3 
https://www.npmjs.com/package/react-native-cardview
Feel free to reopen the issue if not working.
